electron/renderer/src/components/Sidebar.js

```diff
@@ -66,24 +80,51 @@ const Sidebar = ({
         </div>
       </div>
     ))}
-    {!isAddingAccount && !hasReachedLimitOfAccounts &&
-      <ContextMenuTrigger id="account">
-        <div className="Sidebar-cell">
-          <div data-uie-name="do-open-plus-menu" className="Sidebar-account-add">
-            <svg width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg">
-              <path d="M0 5.25v1.5h5.25V12h1.5V6.75H12v-1.5H6.75V0h-1.5v5.25" fillRule="evenodd"/>
-            </svg>
-          </div>
-        </div>
-      </ContextMenuTrigger>
-    }
+    {!isAddingAccount &&
+      !hasReachedLimitOfAccounts && (
+        <ContextMenuTrigger
+          id="account"
+          onClick={preventFocus(event => {
+            const cRect = event.target.getBoundingClientRect();
+            connected.toggleAccountContextVisibility(
+              cRect.left + cRect.width / 2,
+              cRect.top + cRect.height / 2
+            );
+          })}
+          forceVisible={isContextMenuVisible}
+        />
+      )}
 
     <ContextMenu id="account">
-      <ContextMenuItem onClick={() => window.open('https://wire.com/create-team/?pk_campaign=client&pk_kwd=desktop')}>
+      <ContextMenuItem
+        onClick={() =>
+          window.open(
+            'https://wire.com/create-team/?pk_campaign=client&pk_kwd=desktop'
```

Comment:
We need to make this URL environment aware if we have the time to. Code style of these lines also looks off.

Reply:
Let's do this in another PR.